code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -417,76 +421,145 @@
 				if(victim.h_style == "Bald") //you can't scalp someone with no hair.
-					to_chat(user, SPAN_WARNING("You make some rough cuts on [victim]'s head and face with \the [src]."))
+					user.visible_message(SPAN_DANGER("<B>[user] makes some rough cuts on [victim]'s head and face with \a [tool].</B>"),
+						SPAN_DANGER("<B>You make some rough cuts on [victim]'s head and face.</B>"))
 				else
-					to_chat(user, SPAN_WARNING("You use \the [src] to cut around [victim]'s hairline, then rip \his scalp from \his head."))
+					user.visible_message(SPAN_DANGER("<B>[user] cuts around [victim]'s hairline, then tears \his scalp from \his head!</B>"),
+						SPAN_DANGER("<B>You cut around [victim]'s hairline, then rip \his scalp from \his head.</B>"))
 					var/obj/item/scalp/cut_scalp = new(get_turf(user), victim, user) //Create a scalp of the victim at the user's feet.
 					user.put_in_inactive_hand(cut_scalp) //Put it in the user's offhand if possible.
 					victim.h_style = "Bald"
 					victim.update_hair() //tear the hair off with the scalp
-			playsound(loc, 'sound/weapons/slashmiss.ogg', 25)
-
-			if(do_after(user, 4 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, victim))
-				to_chat(user, SPAN_WARNING("You jab \the [src] into the flesh cuts, using them to tear off most of the skin, the remainder skin hanging off the flesh."))
-				playsound(loc, 'sound/weapons/bladeslice.ogg', 25)
-				create_leftovers(victim, has_meat = FALSE, skin_amount = 3)
-				for(var/limb in victim.limbs)
-					victim.apply_damage(18, BRUTE, limb, sharp = FALSE)
-				victim.remove_overlay(UNDERWEAR_LAYER)
-				victim.f_style = "Shaved"
-				victim.update_hair() //then rip the beard off along the skin
-				victim.add_flay_overlay(stage = 2)
-
-				if(do_after(user, 4 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, victim))
-					to_chat(user, SPAN_WARNING("You completely flay [victim], sloppily ripping most remaining flesh and skin off the body. Use rope to hang them from the ceiling."))
-					playsound(loc, 'sound/weapons/wristblades_hit.ogg', 25)
-					create_leftovers(victim, has_meat = TRUE, skin_amount = 2)
-					for(var/limb in victim.limbs)
-						victim.apply_damage(22, BRUTE, limb, sharp = FALSE)
-					for(var/obj/item/item in victim)
-						victim.drop_inv_item_to_loc(item, victim.loc, FALSE, TRUE)
-
-					victim.status_flags |= PERMANENTLY_DEAD
-					victim.add_flay_overlay(stage = 3)
+
+		if(FLAY_STAGE_STRIP)
+			user.visible_message(SPAN_DANGER("<B>[user] jabs \his [tool.name] into [victim]'s cuts, prying, cutting, then tearing off large areas of skin. The remainder hangs loosely.</B>"),
+				SPAN_DANGER("<B>You jab your [tool.name] into [victim]'s cuts, prying, cutting, then tearing off large areas of skin. The remainder hangs loosely.</B>"))
+			playsound(user.loc, 'sound/weapons/bladeslice.ogg', 25)
+			create_leftovers(victim, has_meat = FALSE, skin_amount = 3)
+			flaying_stage = FLAY_STAGE_SKIN
+			for(var/L in victim.limbs)
```

Comment:
you're changing this from a detailed variable name to a single variable name??

Reply:
replacing L to limb like it was before the PR.

Reply:
change this back to a detailed variable name